app/src/main/res/drawable/custom_seekbar.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
What if we used the existing seekbar without inventing a new one?

Reply:
I made this seekbar because the existing seek bar doesnt look good with whole screen , it only suits better with smaller card views.